### DISCUSSION TITLE
Background blur performance causes a spike on transition to gameplay

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15132] [Q&A]
Each time background blur is applied/removed, the game noticeably stutters. It is most noticeable on transition to song select and during the transition to gameplay.

Here is a clip of the issue:

https://user-images.githubusercontent.com/53346335/203297775-f8847f67-5be0-4e9b-a268-7eb14d8e017d.mp4

Logs:
[runtime.log](https://github.com/ppy/osu/files/7357596/runtime.log)
[performance.log](https://github.com/ppy/osu/files/7357597/performance.log)
[performance-update.log](https://github.com/ppy/osu/files/7357595/performance-update.log)
[performance-audio.log](https://github.com/ppy/osu/files/7357598/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7357599/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/7357600/performance-input.log)

**Comment by bdach** [comment]
the blur effect needs work performance-wise, correct. it's been a bit of a forever issue if you will. it's been improved once by using a lower-resolution image if i remember correctly but it's still not 100% where it should be.

**Reply by smoogipoo** [reply]
Even so, this is still much lower performance than I'd expect from a 2017 GPU. Perhaps some special edge case where this GPU is exceptionally terrible at blurs.
